core/parameters/src/config_store.rs

```diff
@@ -50,6 +50,8 @@ static CONFIG_DIFFS: &[(ProtocolVersion, &str)] = &[
     // Increase main_storage_proof_size_soft_limit
     (72, include_config!("72.yaml")),
     (129, include_config!("129.yaml")),
+    // StateStoredReceipt
+    (146, include_config!("146.yaml")),
```

Comment:
Ah just a force of habit to first include new features in nightly before stabilizing them. I think it would be fine to stabilize it immediately here. The important piece is that it's separate from the upcoming gas cost changes. I'll add a comment to that extent and sync with relevant people offline.